collective/dexteritytextindexer/indexer.py

```diff
@@ -33,6 +34,12 @@ def __init__(self, context, request):
 def dynamic_searchable_text_indexer(obj):
     """Dynamic searchable text indexer.
     """
+    # if the object does not provide a request, get one.
+    # This happens when running scripts (bin/instance run script.py)
+    try:
+        obj.REQUEST
+    except AttributeError:
+        obj.REQUEST = getRequest()
```

Comment:
This has side effects: the acquisition wrapper of the object is patched.
Why not always use `getRequest` instead of `obj.REQUEST` below for the `FakeView`?